test/bugs/apps_579_boolean_flag_expr.wdl

```diff
@@ -0,0 +1,25 @@
+version 1.1
```

Comment:
@jdidion   Should I actually make these 1.0 tests? Do we only test against 1.1 for new features not existing in 1.0?

Reply:
From comment on other PR -- 1.1 is ok